dotcom-rendering/src/web/browser/islands/doHydration.tsx

```diff
@@ -23,11 +23,11 @@ export const doHydration = async (
 	data: { [key: string]: unknown } | null,
 	element: HTMLElement,
 	emotionCache: EmotionCache,
-): Promise<void> => {
+): Promise<{ success: boolean }> => {
```

Comment:
`{success: false }` means hydration did not happen because the lightbox has already been hydrated. In this case we don't need to dispatch a synthetic click event, the real one will work